REVIEWING.md

```diff
@@ -0,0 +1,43 @@
+# Pull Request Reviewing Guidelines for Golang Driver Documentation
```

Comment:
C: This should be for Kotlin Sync Driver rather than Golang

```suggestion
# Pull Request Reviewing Guidelines for Kotlin Sync Driver Documentation
```

Reply:
The pitfalls of copy pasting.